android-sdk-restclient/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
     <uses-sdk
         android:minSdkVersion="21"
         android:targetSdkVersion="25" />
-
+    <uses-permission android:name="android.permission.INTERNET"></uses-permission>
```

Comment:
><uses-permission android:name="android.permission.INTERNET"></uses-permission> [](start = 4, length = 78)

nit: consistency with the other uses tag. do: 
<uses-permission
       android:name="android.permission.INTERNET" /> instead of opening and closing, also follow the same pattern with the tag attributes.